FluentTerminal.App/Views/MainPage.xaml

```diff
@@ -56,7 +56,14 @@
                         </MenuFlyoutItem>
                         <MenuFlyoutItem
                             Command="{x:Bind ViewModel.AddSshShellCommand}"
-                            x:Uid="NewSshTab">
+                            x:Uid="NewRemoteTab">
```

Comment:
this has been renamed to NewSshTab with #410

Reply:
@felixse , can you please fix these names as you want them to be? I don't understand...

Reply:
Ahh, OK, i'll rename it back.

Reply:
Fixed.